assembl/static2/js/app/components/administration/saveButton.jsx

```diff
@@ -215,12 +237,33 @@ const SaveButton = ({
           displayAlert('danger', `${error}`, false, 30000);
         });
     }
+
+    if (sectionsHaveChanged) {
+      const mutationsPromises = getMutationsPromises({
+        items: sections,
+        variablesCreator: createVariablesForSectionMutation,
+        deleteVariablesCreator: createVariablesForDeleteSectionMutation,
+        createMutation: createSection,
+        updateMutation: updateSection,
+        deleteMutation: deleteSection
+      });
+
+      runSerial(mutationsPromises)
+        .then(() => {
+          refetchSections();
+          displayAlert('success', I18n.t('administration.sections.successSave'));
+        })
+        .catch((error) => {
+          displayAlert('danger', `${error}`, false, 30000);
```

Comment:
I wonder if I hadn't changed the timeout value to `30000` for debugging purpose and made a mistake by commited it.

Reply:
well it seems not according to blame tab

Reply:
So the error alert is shown for 30s? It's a bit much.

Reply:
no I don't think so. This delay is necessary to do a screenshot of the error for example.